solutions/gold/usaco-1211.mdx

```diff
@@ -21,13 +21,13 @@ Maybe there exists a workaround that can allow us to establish less edges at beg
 
 <Spoiler title="Hint 3">
 
-Maybe we can somehow exploit the y constraint $0 \le y_i \le 10$ given to each cow?
+Given the coordinates of three cows. Can we somehow use the triangle inequality theorem to our advantage to filter out an edge?
 
 </Spoiler>
 
 <Spoiler title="Hint 4">
 
-Given the coordinates of three cows. Can we somehow use the triangle inequality theorem to our advantage to filter out an edge?
+Maybe we can somehow exploit the y constraint $0 \le y_i \le 10$ given to each cow?
```

Comment:
this should be the first one imo

Reply:
like this?

MST > exploit y constraint > explain how the workaround work > triangle inequality

Reply:
oh ok